viewing_party/party.py

```diff
@@ -1,23 +1,201 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    """
+    Sets up a dictionary for a new movie that contials title, genre, and rating. 
+    """   
+    new_movie = {}
+    if not (title and genre and rating):
+        return None 
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating 
+    return new_movie    

```

Comment:
This is nice clean code! :D

As a totally optional refactoring, if I wonder if you all like this alternative?

```python
if not (title and genre and rating):
    return None
    new_movie = {
        "title": title,
        "genre": genre,
        "rating": rating
    }
    return new_movie   
```